store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9379,13 +9379,14 @@ TODO: delete them permanently from here
 </attr>
 
 <attr id="2118" name="zimbraNetworkMobileNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.0" >
-  <globalConfigValue>TRUE</globalConfigValue>
+  <globalConfigValue>FALSE</globalConfigValue>
   <desc>Whether to enable zimbra network new generation mobile sync module.</desc>
 </attr>
 
-<attr id="2119" name="zimbraNetworkAdminEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.2">
-  <globalConfigValue>TRUE</globalConfigValue>
+<attr id="2119" name="zimbraNetworkAdminEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.2" deprecatedSince="8.8.5">
+  <globalConfigValue>FALSE</globalConfigValue>
```

Comment:
Minor: no need to change default value as it is deprecated.

Reply:
Fixed